packages/kbn-eslint-plugin-eslint/rules/no_restricted_paths.js

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable @kbn/eslint/require-license-header */
+/* @notice
```

Comment:
Since we don't ship this code in the distributable I don't think we need to include the `@notice`, do you agree @epixa?

Reply:
Hm, I'd have to reacquaint myself with how we handle that annotation. I don't think it makes sense to ship this package's license info in our distributions' notice file. I do still think we should be automatically checking it's license, albeit against our dev-acceptable list. In general, I'd rather ship a dependency's info in our notice file by accident than accidentally leave out a dependency we should have included.

Reply:
This tag doesn't cause the licence to be checked, that's something we do "manually" by committing the code to the repo. This tag is only for generating the notice file in the build, so I think we can get rid of it.